src/main/resources/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "rdp",
+  "private": true,
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "dependencies": {
+    "bootstrap": "4.0.0",
```

Comment:
We can do some pruning here by selecting parts of bootstrap to import.

Reply:
It's too complicated to do right now, we have to use https://github.com/webpack-contrib/exports-loader to make this work.